demo/owner_alice/create_layout.py

```diff
@@ -53,9 +55,9 @@ def main():
             ["ALLOW", "alice.pub"],
             ["ALLOW", "root.layout"],
         ],
-        "product_matchrules": [
+        "expected_products": [
             ["MATCH", "demo-project/foo.py", "WITH", "PRODUCTS", "FROM", "update-version"],
-            # FIXME: See material_matchrules above
+            # FIXME: See expected_materials above
```

Comment:
Actually this one is still relevant, because it refers to the `FIXME` in line 51, which we have not fixed yet. But the one in 29 has been fixed. :)